source/guides/references/configuration.md

```diff
@@ -14,15 +14,88 @@ The default behavior of Cypress can be modified by supplying any of the followin
 
 ## Global
 
-Option | Default | Description
------ | ---- | ----
-`baseUrl` | `null` | URL used as prefix for {% url `cy.visit()` visit %} or {% url `cy.request()` request %} command's URL
-`env` | `{}` | Any values to be set as {% url 'environment variables' environment-variables %}
-`numTestsKeptInMemory` | `50` | The number of tests for which snapshots and command data are kept in memory. Reduce this number if you are experiencing high memory consumption in your browser during a test run.
-`port` | `null` | Port used to host Cypress. Normally this is a randomly generated port
-`reporter` | `spec` | The {% url 'reporter' reporters %} used during `cypress run`
-`reporterOptions` | `null` | The {% url 'reporter options' reporters#Reporter-Options %} used. Supported options depend on the reporter.
-`watchForFileChanges` | `true` | Whether Cypress will watch and restart tests on test file changes
+The following options affect the running Cypresss instance globally. Refer to the examples below for different ways to apply these options. 
+
+## `baseUrl`
+URL used as prefix for {% url `cy.visit()` visit %} or {% url `cy.request()` request %} command's URL
+
+**Default value:** `null`
+
+**Example:** pass staging url during tests by overriding ENV variable
```

Comment:
These headings aren't consistent with the rest of our documentation - there should be a level `###` heading so as to enable linking. Showing the default value like this is also unconventional. I'd have to look at other examples of default values and edit this to be consistent.